tests/test_claude_wrapper_tmux_env_regression.py

```
@@ -0,0 +1,188 @@
+
+
+def expect(condition: bool, message: str, failures: list[str]) -> None:
+    if not condition:
+        failures.append(message)
+
+
+def test_tmux_like_env_injects_hooks_without_surface(failures: list[str]) -> None:
+    code, argv, claudecode, stderr, _debug_lines = run_wrapper(
+        ["chat", "hello"],
+        {
+            "TMUX": "/tmp/tmux-1000/default,1,0",
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/tmux-1000/default,1,0"

(S108)